web/pdf_presentation_mode.js

```diff
@@ -97,14 +97,14 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * Switches page when the user scrolls (using a scroll wheel or a touchpad)
      * with large enough motion, to prevent accidental page switches.
-     * @param {number} delta - The delta value from the mouse event.
+     * @param {number} delta - The value from the mouse event in page units.
      */
     mouseScroll: function PDFPresentationMode_mouseScroll(delta) {
       if (!this.active) {
         return;
       }
       var MOUSE_SCROLL_COOLDOWN_TIME = 50;
-      var PAGE_SWITCH_THRESHOLD = 120;
+      var PAGE_SWITCH_THRESHOLD = 1.0;
```

Comment:
Yeah, I realized that this could become problematic given that it might depend on the configuration.
One thing I'm wondering though, since it seems to fix things for me, would it work if we simply [changed this line](https://github.com/mozilla/pdf.js/pull/7181/files#diff-529d1853ee1bba753a0fcb40ea778723R1845) back to `PDFViewerApplication.scrollPresentationMode(delta * MOUSE_WHEEL_ZOOM_FACTOR);` instead?